apps/tenant-management-webapp/src/app/store/configuration/sagas.ts

```diff
@@ -126,7 +132,59 @@ export function* fetchConfigurations(action: FetchConfigurationsAction): SagaIte
     }
   }
 }
+export function* fetchConfigurationRevisions(action: FetchConfigurationRevisionsAction): SagaIterator {
+  yield put(
+    UpdateIndicator({
+      show: true,
+      message: 'Loading...',
+    })
+  );
+  const configBaseUrl: string = yield select(
+    (state: RootState) => state.config.serviceUrls?.configurationServiceApiUrl
+  );
+  const token: string = yield call(getAccessToken);
+  const service = action.service.split(':');
+  const url = `${configBaseUrl}/configuration/v2/configuration/${service[0]}/${service[1]}/revisions?top=10&after=${
+    action.after || ''
+  }`;
+  if (configBaseUrl && token) {
+    try {
+      const { data } = yield call(axios.get, url, { headers: { Authorization: `Bearer ${token}` } });
+
+      yield put(getConfigurationRevisionsSuccess(data.results, action.service, data.page.after, data.page.next));
+      yield put(
+        UpdateIndicator({
+          show: false,
+        })
+      );
+    } catch (err) {
+      yield put(ErrorNotification({ message: err.message }));
+      yield put(
+        UpdateIndicator({
+          show: false,
+        })
+      );
+    }
+  }
+}
 
+export function* fetchConfigurationActiveRevision(action: FetchConfigurationActionRevisionAction): SagaIterator {
+  const configBaseUrl: string = yield select(
+    (state: RootState) => state.config.serviceUrls?.configurationServiceApiUrl
+  );
+  const token: string = yield call(getAccessToken);
+  const service = action.service.split(':');
+  const url = `${configBaseUrl}/configuration/v2/configuration/${service[0]}/${service[1]}/active`;
+  if (configBaseUrl && token) {
+    try {
+      const { data } = yield call(axios.get, url, { headers: { Authorization: `Bearer ${token}` } });
+
+      yield put(getConfigurationActiveSuccess(data, action.service));
+    } catch (err) {
+      yield put(getConfigurationActiveSuccess(null, action.service));
```

Comment:
It looks like we are kind of making the exception too slience. We can improve it later.

Reply:
Exception means no active revision.